.tours/2---object agnostic design.tour

```diff
@@ -0,0 +1,53 @@
+{
+  "title": "2 - Object-Agnostic Design. We use custom metadata to reuse Lightning Web Components and Flow Apex Actions in any object without having to make any code changes",
+  "steps": [
+    {
+      "file": "es-space-mgmt/main/default/lwc/customerList/customerList.html",
+      "description": "The `customerList` Lightning Web Component can display information from any object if the object to display is configured in the Custom Metadata **Customer Fields**",
+      "line": 14
+    },
+    {
+      "file": "es-space-mgmt/main/default/lwc/customerList/customerList.js",
+      "description": "The `customerList` component controller takes the object name as an input parameter and passes to the apex controller method, which then can query a custom metadata record with the object name",
```

Comment:
Simplify to say, the customerList component passes the sobject as a param to an Apex method using @wire. The sobject param is supplied using the App Builder. 
After this step, show the meta file.
Then show the Apex class - in which you highlight the query
Then show the object fields.

Reply:
fixed!